install_config/provisioners.adoc

```diff
@@ -22,7 +22,6 @@ independently of {product-title}.
 
 [[provisioners-before-you-begin]]
 == Before You Begin
-An link:https://github.com/openshift/openshift-ansible/blob/master/playbooks/openshift-provisioners/config.yml[Ansible Playbook] is also available to deploy and upgrade external provisioners.
```

Comment:
@gpei Can we keep this link in OpenShift origin?

Reply:
Not sure about this, we didn't use or test Origin.